src/BuildScriptGeneratorCli/ServiceProviderBuilder.cs

```diff
@@ -65,24 +86,6 @@ public IServiceProvider Build()
         private static LoggingConfiguration BuildNLogConfiguration([CanBeNull] string logPath)
         {
             var config = new LoggingConfiguration();
```

Comment:
Added this method back here so we can continue to use NLog to write telemetry to a .txt file. 
Spent some time experimenting with Serilog and Telemetry Channel (can be found [here](https://learn.microsoft.com/en-us/azure/azure-monitor/app/telemetry-channels#configuration-in-code-for-aspnet-applications)). The latter proved to be not that useful because it only provides a temporary storage for logs that were not sent to AI.
With Serilog however, we need to make some tweaks to make sure the log format doesn't change.
Since we need a third party package anyway for this use case, I think it is best we stick to NLog to write log telemetry to a text file. I was also able to verify that there is no change to the log format if continue to use NLog.